### PR TITLE
[NUI] Add SelectionStarted event

### PR DESCRIPTION
**Pull request by abdullehghujeh**
### Description of Change ###
Add SelectionStarted event

### API Changes ###
editor.SelectionStarted += (s, e) =>
{
};

### CODE REVIEW

**Review by wonrst — COMMENTED** [review]
Hello @abdullehghujeh 
It works fine :) But how about adding a test?
Please check my comment.

**Review by JoogabYun — APPROVED** [review]

File: test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/BaseComponents/TSTextEditorEvent.cs [hunk 1]
```diff
@@ -51,6 +51,9 @@ public void TextEditorEventEvents()
             testingTarget.AnchorClicked += OnAnchorClicked;
             testingTarget.AnchorClicked -= OnAnchorClicked;
 
+            testingTarget.SelectionStarted += OnSelectionStarted;
```
[the remark this answers is not in the document]
Comment (abdullehghujeh) [reply test/Tizen.NUI.Tests/Tizen.NUI.Devel.Tests/testcase/public/BaseComponents/TSTextEditorEvent.cs:54]: so I should move this test from current path to this path `TizenFX/test/Tizen.NUI.Devel.Tests.Ubuntu/Tizen.NUI.Devel.Tests/` and update pull request ?
or you mean as a help only to run it & test if it is working ?

**Review by Seoyeon2Kim — APPROVED** [review]
LGTM

**Review by wonrst — APPROVED** [review]
Looks good and related patches have been merged.
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-toolkit/+/269516/
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/269643/

**Review by everLEEst — APPROVED** [review]
Looks good to me.

**Pull request merged**